catalog/plugin/catalog-plugin-metacard-validation/pom.xml

```diff
@@ -84,7 +84,7 @@
                                         <limit>
                                             <counter>INSTRUCTION</counter>
                                             <value>COVEREDRATIO</value>
-                                            <minimum>0.9</minimum>
+                                            <minimum>0.8</minimum>
```

Comment:
Jacoco coverage checks do pass with  the defaults so I would be ok with removing it.  Still have the questions about groupId/name